reason-react-native/src/components/FlatList.md

```diff
@@ -25,19 +25,15 @@ external make:
     ~getItemLayout: (array('item), int) => VirtualizedList.itemLayout=?,
     ~numColumns: 'int=?,
     // VirtualizedList props
-    // ~_CellRendererComponent: VirtualizedList.cellRendererComponent('item)=?,
+    ~_CellRendererComponent: VirtualizedList.cellRendererComponent('item)=?,
     ~_ListEmptyComponent: unit => React.element=?,
     ~_ListFooterComponent: unit => React.element=?,
     ~_ListFooterComponentStyle: Style.t=?,
     ~_ListHeaderComponent: unit => React.element=?,
     ~_ListHeaderComponentStyle: Style.t=?,
     ~debug: bool=?,
-    // ~data: 'data, // any collection of 'item
```

Comment:
`~data` was defined twice and the compiler did not notice? I thought we had all/most warnings enabled?

Reply:
defined but commented. It was to keep list complete so we don't mess up when syncing `// VirtualizedList props`.
I think we should keep this things commented & maybe add another comment on top if we really want an explanation

Reply:
Ah yes, it was commented out. 🤦‍♂️ Sorry!